tests/pytests/functional/modules/test_cmdmod.py

```diff
@@ -0,0 +1,564 @@
+import os
+import random
+import sys
+from contextlib import contextmanager
+
+import pytest
+import salt.config
+import salt.utils.path
+import salt.utils.platform
+import salt.utils.user
+from tests.support.helpers import SKIP_INITIAL_PHOTONOS_FAILURES, dedent
+
+pytestmark = [pytest.mark.windows_whitelisted]
+
+
+@pytest.fixture(scope="module")
+def cmdmod(modules):
+    return modules.cmd
+
+
+@pytest.fixture(scope="module")
+def usermod(modules):
+    return modules.user
+
+
+@pytest.fixture(scope="module")
+def available_python_executable():
+    yield salt.utils.path.which_bin(["python", "python2", "python2.6", "python2.7"])
```

Comment:
```suggestion
    yield salt.utils.path.which_bin(["python", "python3"])
```
Let's stop searching for python 2